src/screens/Layout/index.tsx

```diff
@@ -14,6 +15,14 @@ function Layout({ children }: Props) {
     <div className={styles.layout}>
       <Toaster position="top-right" />
       {/* Future Header */}
+      <nav>
+        <NavLink className={({ isActive }) => (isActive ? styles.active : '')} to="/">
+          Login
+        </NavLink>
+        <NavLink className={({ isActive }) => (isActive ? styles.active : '')} to="/sign_up">
+          Sign up
+        </NavLink>
```

Comment:
This link should be the buttons of the containers, not a navbar

Reply:
Removed temporary nav and added links on buttons :suspect: